src/pages/form/advanced-form/index.tsx

```diff
@@ -0,0 +1,308 @@
+import { Button, Card, Col, DatePicker, Input, Popover, Row, Select, TimePicker } from 'antd';
+import React, { Component } from 'react';
+
+import { Dispatch } from 'redux';
+import { FormComponentProps } from 'antd/es/form';
+import { Icon as LegacyIcon, Form } from '@ant-design/compatible';
+import { PageHeaderWrapper, RouteContext } from '@ant-design/pro-layout';
+import { connect } from 'dva';
+import TableForm from './components/TableForm';
+import FooterToolbar from './components/FooterToolbar';
+import styles from './style.less';
+
+const { Option } = Select;
+const { RangePicker } = DatePicker;
+
+const fieldLabels = {
+  name: '仓库名',
+  url: '仓库域名',
+  owner: '仓库管理员',
+  approver: '审批人',
+  dateRange: '生效日期',
+  type: '仓库类型',
+  name2: '任务名',
+  url2: '任务描述',
+  owner2: '执行人',
+  approver2: '责任人',
+  dateRange2: '生效日期',
+  type2: '任务类型',
+};
+
+const tableData = [
+  {
+    key: '1',
+    workId: '00001',
+    name: 'John Brown',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'name' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)